### DISCUSSION TITLE
Wezterm version as environmental variable

### DISCUSSION THREAD

**Discussion by Lilja** [wezterm/wezterm#4163] [Ideas]
I sometimes use wezterm on my pc through WSL(Windows Subsystem for Linux), and get the notification that there is a new release for wezterm. In those times I would like to know what version I'm running. I can find this information out through going to the programs list(Win key + "Programs"  > "Apps & features" > search for wezterm > seeing the release information there). It works but I would much rather have something within wezterm that I can see that quicker.

What if there was a environmental variable set so I could `$ env | grep WEZ` in my shell and see that version?

**Comment by nekowinston** [comment]
`wezterm --version` is supposed to display version information, if you really need an environment variable, you could do
```sh
if [[ "$TERM_PROGRAM" == "WezTerm" ]]; then
  WEZTERM_VERSION="$(wezterm --version)"
fi
```

**Reply by wez** [reply]
wezterm sets `TERM_PROGRAM` to `WezTerm` and `TERM_PROGRAM_VERSION` to the version of wezterm.

**Comment by wez** [comment]
Also note: you can press CTRL-SHIFT-L to open the debug overlay and see the version of wezterm
